Localization/es-MX/faq.po

```diff
@@ -17,7 +17,7 @@ msgstr ""
 msgid "FAQ"
 msgstr "PF"
 
-msgid "You'll find a large library of frequently asked questions about all things connected to Folding@home."
+msgid "**You'll find a large library of frequently asked questions about all things connected to Folding@home.**"
 msgstr "Encontrarás una gran biblioteca de preguntas frecuentes sobre todo lo relacionado con Folding@home."
```

Comment:
Stars at the beginning and end are missing inside msgstr

Reply:
Done.